tests/integration/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/skip-mev/connect/tests/integration/v2
+module github.com/skip-mev/connect/v2/tests/integration
```

Comment:
they way it was previously made more sense to me. any reason to change this here?

Reply:
this way seems to not work.  When adding new module to neutron:

```
go: finding module for package github.com/skip-mev/connect/tests/integration/v2
go: found github.com/skip-mev/connect/tests/integration/v2 in github.com/skip-mev/connect/tests/integration/v2 v2.0.0
go: neutron/tests/connect tested by
	neutron/tests/connect.test imports
	github.com/skip-mev/connect/tests/integration/v2: github.com/skip-mev/connect/tests/integration/v2@v2.0.0: parsing go.mod:
	module declares its path as: github.com/skip-mev/connect/v2/tests/integration/v2
	        but was required as: github.com/skip-mev/connect/tests/integration/v2
```